java/custom/java/lang/String.java

```diff
@@ -197,7 +200,7 @@ public String(char value[], int offset, int count) {
      * @since      JDK1.1
      */
     public String(byte bytes[], int off, int len, String enc)
-        throws UnsupportedEncodingException
+            throws UnsupportedEncodingException
```

Comment:
Could you avoid indentation changes in these files? It makes it easier to review and, in the future, to see what we've changed w.r.t. the reference implementation